update-resolv-conf.sh

```diff
@@ -17,7 +17,11 @@
 # foreign_option_3='dhcp-option DOMAIN be.bnc.ch'
 # foreign_option_4='dhcp-option DOMAIN-SEARCH bnc.local'
 
-## You might need to set the path manually here, i.e.
+## The 'type' builtins will look for file in $PATH variable, so we set the
+## PATH below. You might need to directly set the path to 'resolvconf'
+## manually if it still doesn't work, i.e.
+## RESOLVCONF=/usr/sbin/resolvconf
+export PATH=$PATH:/sbin:/usr/sbin:/bin:/usr/bin
```

Comment:
The thought originally came from my local bash configuration. 'export'ing a variable can make it available to other programs, a quick ref:

- http://stackoverflow.com/questions/7411455/what-does-export-do-in-shell-programming

I tried setting the 'PATH' var without exporting it in the 'openvpn-update-resolv-conf' script just now, it also worked without problem, as far as I could see the 'resolvconf' script can work without throwing errors. We can go ahead to remove the 'export' directive.